internal/gensupport/media_test.go

```diff
@@ -310,11 +310,11 @@ func TestUploadRequestGetBody(t *testing.T) {
 		},
 		{
 			desc: "chunk size < data size: MediaBuffer, >1 chunk, no getBody",
```

Comment:
This should be "getBody" rather than "no getBody". Also, comment on L313 can be removed.